src/spaceone/dashboard/service/project_dashboard_service.py

```diff
@@ -50,7 +50,8 @@ def create(self, params):
         project_dashboard_vo = self.project_dashboard_mgr.create_project_dashboard(params)
 
         version_keys = ['layouts', 'dashboard_options', 'dashboard_options_schema']
-        if set(version_keys) <= params.keys():
+        if any(key for key in params
```

Comment:
I suggest improving the code here as follows.
any(set(version_keys) & set(params.keys()))